webapp/src/utils/jira_issue_metadata.tsx

```diff
@@ -55,7 +55,7 @@ export function getProjectValues(metadata: ProjectMetadata | null): ReactSelectO
     return metadata.projects;
 }
 
-export function getIssueTypes(metadata: IssueMetadata | null, projectKey: string | null): IssueType[] {
+export function getIssueTypes(metadata: IssueMetadata | null, projectKey: string | null, includeSubtasks: boolean): IssueType[] {
```

Comment:
Just to make this easier to read on the caller's side

```suggestion
type GetIssueTypesOptions = {
    includeSubtasks: boolean;
}

export function getIssueTypes(metadata: IssueMetadata | null, projectKey: string | null, options: GetIssueTypesOptions): IssueType[] {
```